templates/org/home.tmpl

```diff
@@ -24,6 +24,31 @@
 					</div>
 					<div class="divider"></div>
 				{{end}}
+
+				{{if and .ShowMemberAndTeamTab .HasPublicProfileReadme .HasPrivateProfileReadme}}
+				<div class="tw-my-4">
+					<div id="org-home-view-as-dropdown" class="ui dropdown jump">
+							<span class="text">
+								{{svg "octicon-eye"}}
+								View as: {{if not .IsViewerMember}}{{ctx.Locale.Tr "settings.visibility.public"}}{{else}}{{ctx.Locale.Tr "org.members.member"}}{{end}}
+							</span>
+						{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						<div class="menu">
+							{{/* TODO: does it really need to use CurrentURL with query parameters? Why not construct a new link with clear parameters */}}
+							<a href="{{QueryBuild $.CurrentURL "view_as" "public"}}" class="item {{if not .IsViewerMember}}selected{{end}}">
+								{{svg "octicon-check" 14 (Iif (not .IsViewerMember) "" "tw-invisible")}} {{ctx.Locale.Tr "settings.visibility.public"}}
+							</a>
+							<a href="{{QueryBuild $.CurrentURL "view_as" "member"}}" class="item {{if .IsViewerMember}}selected{{end}}">
+								{{svg "octicon-check" 14 (Iif .IsViewerMember "" "tw-invisible")}}  {{ctx.Locale.Tr "org.members.member"}}
+							</a>
```

Comment:
This is copied from your code (user current request's query parameters)

But I am not sure why we should do it, why it needs to keep all the query parameters but not generate a new link with clear parameters?

Reply:
I wanted to make `view_as` `sort` and `language` work at the same time. But I just noticed Github reset query params when view_as is applied.

Reply:
Then let's keep it simple, I do not see it is useful to keep other parameters.